apps/devhub/tasks.py

```diff
@@ -180,13 +181,34 @@ def handle_upload_validation_result(results, upload_pk, annotate=True):
     now = datetime.datetime.now()
     now_ts = utc_millesecs_from_epoch(now)
     delta = now_ts - upload_start
+    statsd.timing('devhub.validation_results_processed', delta)
+
+    size = Decimal(storage.size(upload.path))
+    megabyte = Decimal(1024 * 1024)
+
+    if size <= megabyte:
+        # Measure small files in a separate bucket.
+        statsd.timing('devhub.validation_results_processed_under_1mb', delta)
```

Comment:
I guess it wouldn't hurt to add it. I can't find in the docs how to subtract metrics in data dog :fries: but you could do it in graphite.